### ISSUE TITLE
Migrate Wiki

### ISSUE THREAD

**Issue by indolering** [namecoin/namecoin-legacy#35]
https://github.com/namecoin/wiki/wiki

Here is the situation:  The wiki was overflowing with spam and Khal moved it to Github.  I created a list of the top articles and Ademan used that list as a seed to spider the old wiki and then pushed it to github.com/namecoin/wiki/.  Sadly, the Github wiki does not allow for per-page permission settings, meaning that we can't (for example) lock down pages which link to binary downloads or NMC/BTC donation addresses.  So we will have to push things out to a new Mediawiki install after we get the Namecoin hosting sorted out (~1-2 weeks).

However, the wiki needed an overhaul anyway.  There were many old TODO pages, forgotten proposals and outdated lists, 90% of which has been deleted.  The last remaining items needing to be cleaned, sorted and merged are:

Standard access
~~How To Browse .bit Domains~~
~~HowToBrowseBitDomainsAdvanced~~
Hold until new wiki.

Alternatives
How To Configure Dns Suffix
DNS Translation and Proxy Software 
List Of Web Proxies

List of software, grouped by category.
How To Create A Dns
Namecoin-GUI-Info
Namecoin WUI
Any other page that simply lists software.

**Comment by taoeffect** [comment]
This is an awesome list! Thanks for putting this together! :+1:

**Comment by Ademan** [comment]
I still cannot modify namecoin/namecoin/wiki  However I pushed the content I scraped to namecoin/wiki, and then from there to namecoin/wiki/wiki (namecoin/wiki is a plain repository and github won't treat it like a wiki, namecoin/wiki/wiki is the wiki for namecoin/wiki, and contains the same content as namecoin/wiki). I need to ensure the whole list was grabbed, and we need to fix where github's mediawiki parser fails (advanced tables, include directives). In my opinion it's already better than the current state of namecoin/namecoin/wiki and should be pushed, but I don't have sufficient permissions to do that.
